templates/individualDrive.html

```diff
@@ -10,14 +10,20 @@
     </div>
   {% endif %}
 {% endwith %}
+<link rel="stylesheet" href="../static/Styles/create_drive.css">
```

Comment:
this shouldn't go in the body

there should be a 
{% body Head %} tag you can use